compose/samples/src/main/AndroidManifest.xml

```diff
@@ -35,6 +35,7 @@
     <activity android:name=".hellocomposerendering.HelloComposeRenderingActivity" />
     <activity android:name=".nestedrenderings.NestedRenderingsActivity" />
     <activity android:name=".textinput.TextInputActivity" />
+    <activity android:name=".preview.PreviewActivity" />
```

Comment:
nit if you want it: looks like these were alphabetized.

Reply:
- [ ] alphabetize samples